dodo.py

```diff
@@ -0,0 +1,325 @@
+##
+# This file contains various build steps used to make sure the data the Hedy application
+# uses is up-to-date.
+#
+# Functions that start with the name `task_` create a "task", which runs one or more
+# "actions" (shell commands or Python functions). Tasks can have file dependencies and
+# file outputs, as well as other task dependencies. If none of the files that a task
+# depends on have changed since the last time it ran, the task will be skipped. That's
+# good for speed!
```

Comment:
this sounds promising! How does it do that and does it do it with tasks as well? Or it's just with files and tasks

Reply:
Just with files. It does it by calculating and storing the `md5` hash of every file. By doing that, it can check if files have changed, and skip the task if they haven't.